docs/storefront-integration/theme-customization.md

```diff
@@ -18,7 +24,7 @@ Each theme and business is different, you can apply any of these techniques that
 ### Product tagging
 Tagging products will allow you to apply these customizations only on products that are used as a service.
 
-We recommend adding `sesami-service` as a tag on your Sesami services.
+We recommend adding `sesami-service` as a tag on your Sesami services. Sesami is auto-tagging all Shopify products you add to Sesami as a service. 
```

Comment:
Older sentence can be removed completely. or:
Sesami adds `sesami-service` tag to all Shopify products...